iris_grib/__init__.py

```diff
@@ -50,7 +50,7 @@
 __version__ = '0.9.0'
 
 __all__ = ['load_cubes', 'save_grib2', 'load_pairs_from_fields',
-           'save_pairs_from_cube', 'save_messages', 'GribWrapper']
```

Comment:
Do we know why `GribWrapper` was in the public API but `GribMessage` wasn't?

Further, if we've removed it from the public API do we want to mark it as "private" with a leading underscore (_) on the class name?